### DISCUSSION TITLE
Feature Request: Create a document without editing

### DISCUSSION THREAD

**Discussion by wlatic** [outline/outline#1493] [General]
Normally when I'm creating a wiki / knowledgebase I'll do one pass creating structure and then another editing the documents.

It'd be great if there was a way of creating the docs (within the structure) without being sent to the edit page.

**Comment by tommoor** [comment]
Some way you could just type the document title in-line in the sidebar would be nice. Not sure how the best way to trigger that input would be, particularly as the vast majority of the time folks _will_ want to immediately add content to a document, not just the title.

**Reply by wlatic** [reply]
I was thinking a double click on document name? What would be really cool would be a short cut that only worked when over a collection or nested doc

Even if it was in context menu as "New Quick Nested Document" and then the new document was created and you then renamed it on the left it'd save time and allow the "structure" type work flow.

Its really only a nice to have, just started using this a few days ago and its an awesome product.

**Reply by tommoor** [reply]
Double click to rename for sure 🙏 

I think it's a good idea, anything to make workflows faster
